### ISSUE TITLE
Incorrect line:column numbers

### ISSUE THREAD

**Issue by glesaint** [ConsenSys/truffle-security#186]
(Not sure wether this is the same as https://github.com/ConsenSys/truffle-security/issues/66)

In some cases, the line:column numbers reported by truffle run verify do not correspond to actual code locations. 
See this repository to reproduce the issue: https://github.com/glesaint/truffle-security-issue

See a report extract + corresponding source location.
```
./truffle-security-issue/contracts/token/ERC20/TokenTimelock.sol
  54:4     warning  Multiple sends are executed in one transaction  SWC-113
  64:1545  warning  The contract executes an external message call  SWC-107`
```
Lines 54 to 64:
```
function release() public {
        // solhint-disable-next-line not-rely-on-time
        require(block.timestamp >= _releaseTime);

        uint256 amount = _token.balanceOf(address(this));
        require(amount > 0);

        _token.safeTransfer(_beneficiary, amount);
    }
}
```

**Comment by daniyarchambylov** [comment]
@glesaint @b-mueller new version  of truffle-security has been released. The release includes fix which corrects line numbers in reports #185. Could you please share feedback whether issue still present?

**Comment by glesaint** [comment]
@daniyarchambylov, thanks this is resolved with 1.3.6
